modules/oe_paragraphs_timeline/oe_paragraphs_timeline.module

```diff
@@ -26,88 +27,47 @@ function oe_paragraphs_timeline_field_widget_timeline_widget_form_alter(&$elemen
 
   // We can't use the protected isDefaultValueWidget() method.
   $is_default_value_widget = (bool) $form_state->get('default_value_widget');
-  if ($is_default_value_widget) {
-    return;
-  }
 
-  $element['label']['#required'] = $element['#required'];
-  $element['title']['#required'] = $element['#required'];
-  $element['#element_validate'][] = '_oe_paragraphs_validate_required_fields';
-
-  // If the elements were set as required, there is no need to append states.
-  if ($element['#required']) {
+  // If the default value of the widget did not change, we bail out.
+  if ($is_default_value_widget) {
     return;
   }
 
   // Get the selector to build the paths for each field.
+  $field_name = $field_definition->getName();
+  $delta = $element['#delta'];
   $parents = $element['#field_parents'];
-  $parents[] = $field_definition->getName();
-  $parents[] = $element['#delta'];
-  $selector = array_shift($parents);
-  if ($parents) {
-    $selector .= '[' . implode('][', $parents) . ']';
-  }
+  $parents[] = $field_name;
+  $selector = array_shift($parents) . '[' . implode('][', $parents) . ']' . '[' . $delta;
 
-  // Mark label as required when title is filled in and vice-versa.
-  // The content element can have an editor assigned, and states do not play
-  // nicely with it, so we don't show any visual hint and rely only on
-  // server-side validation.
-  foreach (['label', 'title'] as $name) {
-    $element[$name]['#states']['required'][] = [
-      ':input[name="' . $selector . '[title]"]' => ['filled' => TRUE],
```

Comment:
Are you sure all of this needs to be reverted?

Reply:
I'm not sure this is needed because if you fill in content it will set required label and title. If you fill in title => label required but title won't be set required. The same for label.